sky/cli.py

```diff
@@ -1174,7 +1174,7 @@ def launch(
     and they undergo job queue scheduling.
     """
     backend_utils.check_cluster_name_not_reserved(
-        cluster, operation_str='Launching task on it')
+        cluster, operation_str='Launching tasks on it')
```

Comment:
What does `it` refer to?

Reply:
Instead of passing in `operation_str` (adding `is not allowed` may not be general for all reserved clusters), pass in `error_str` with `error_str= Launching task on {cluster} is not allowed`

Reply:
It looks fine to me:
```
» ag -w check_cluster_name_not_reserved -A1
sky/execution.py
359:    backend_utils.check_cluster_name_not_reserved(cluster_name,
360-                                                  operation_str='sky.launch')
--
437:    backend_utils.check_cluster_name_not_reserved(cluster_name,
438-                                                  operation_str='sky.exec')

sky/backends/backend_utils.py
1950:def check_cluster_name_not_reserved(
1951-        cluster_name: Optional[str],

sky/core.py
442:    backend_utils.check_cluster_name_not_reserved(
443-        cluster_name, operation_str='Cancelling jobs')

sky/cli.py
1176:    backend_utils.check_cluster_name_not_reserved(
1177-        cluster, operation_str='Launching tasks on it')
--
1314:    backend_utils.check_cluster_name_not_reserved(
1315-        cluster, operation_str='Executing task on it')
```
as it doesn't repeat "is not allowed". cc @Michaelvll

Reply:
I feel like it would be better to show more detailed information as the `operation_str` does, instead of a general `is not allowed`. It will provide more information for the user as well as the developer for debugging.